jaeger_client/thrift.py

```diff
@@ -90,6 +97,18 @@ def make_tag(key, value, max_length):
         )
 
 
+def _make_traceback_tag(key, value, max_length):
+    key = _to_string(key)
+    value = ''.join(traceback.format_tb(value))
+    if len(value) > max_length:
+        value = value[:max_length]
```

Comment:
Hmm, max_length may be pretty limiting for a stack trace. Would it make sense to have a different parameter controlling max stack trace size?

Reply:
I did notice stack traces getting cut off rather early, so it would definitely make sense to extend that length for stack traces. It seems like a lot would need to be changed in order to add the max stack trace size as a parameter though. I can make those changes if adding such a parameter feels like a good idea. An alternative would be to use a larger constant instead of having it as a parameter.